sdk/netapp/Azure.ResourceManager.NetApp/api/Azure.ResourceManager.NetApp.netstandard2.0.cs

```diff
@@ -1374,24 +1306,21 @@ protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer
         public override string ToString() { throw null; }
     }
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
-    public readonly partial struct ExternalReplicationSetupStatus : System.IEquatable<Azure.ResourceManager.NetApp.Models.ExternalReplicationSetupStatus>
+    public readonly partial struct Exclude : System.IEquatable<Azure.ResourceManager.NetApp.Models.Exclude>
```

Comment:
Single-word names are not permitted. Please rename this enum to a more descriptive name.